From bb87ae14e5a381617d1969837efd0d9c3ddefa25 Mon Sep 17 00:00:00 2001
From: David Grayson <davidegrayson@gmail.com>
Date: Tue, 9 May 2017 09:15:29 -0700
Subject: [PATCH 1/2] specstrings.h:  Include driverspecs.h near the bottom.

That's what the Microsoft header does, and their devcon utility
depends on it.
---
 mingw-w64-headers/include/specstrings.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/mingw-w64-headers/include/specstrings.h b/mingw-w64-headers/include/specstrings.h
index 485b4330..984dd64b 100644
--- a/mingw-w64-headers/include/specstrings.h
+++ b/mingw-w64-headers/include/specstrings.h
@@ -328,3 +328,5 @@ extern "C" {
 #ifdef __cplusplus
 }
 #endif
+
+#include <driverspecs.h>
-- 
2.12.1

